docs/self-managed/reference-architecture/kubernetes.md

```diff
@@ -0,0 +1,287 @@
+---
+id: kubernetes
+title: "Kubernetes deployment overview"
+sidebar_label: "Kubernetes"
+description: "Camunda 8 Kubernetes deployment Reference architecture home"
+---
+
+<!-- TODO: Update for 8.8 changes when it's clear what it means -->
+
+This reference architecture provides guidance on deploying Camunda 8 Self-Managed within a Kubernetes cluster. This deployment method is ideal for users who are looking to leverage the benefits of containerization and self-healing capabilities. It's well suited to quickly scale and easily provide resilience by utilizing multi-zone deployments.
+
+## Key features
+
+- **Scalability & High Availability**: Camunda 8 can scale up and down depending on your demand, and makes use of a highly available setup.
+- **Fault Tolerance & Resilience**: Scale Camunda 8 across zones or across regions to provide greater fault tolerance and uptime for your workflows.
+
+Although Kubernetes has a steeper learning curve, once properly configured, it provides significant advantages, including self-healing capabilities that ensure application resilience by automatically restarting failed containers and rescheduling workloads.
+
+Additionally, Kubernetes benefits from a vast ecosystem of **[Cloud Native Computing Foundation (CNCF) projects](https://www.cncf.io/)**, offering seamless integration with monitoring, observability, logging, and security solutions. Tools like Prometheus and Grafana allow for comprehensive monitoring, while service meshes such as Istio enhance traffic management and security.
+
+By leveraging this ecosystem, organizations can extend Kubernetes functionality to meet their specific operational needs, enabling greater automation, scalability, and visibility across their infrastructure.
+
+## Reference implementations
```

Comment:
I am wondering if this section would be better incorporated elsewhere. Does it make sense to provide these two (and only two) references up front?

Could we to the overview for each cloud provider within the cloud-specific section at the end?

Reply:
I remember for manual I had them at the bottom in the beginning and then they got moved up.

The amount will grow over time, e.g. Azure in the workings atm.

The problem I see with moving them to the cloud-specific part is that you don't immediately see them in the table of contents anymore and people may miss it.